rotkehlchen/premium/sync.py

```diff
@@ -42,7 +43,7 @@ def __init__(self, migration_manager: DataMigrationManager, data: DataHandler) -
         # Initialize this with the value saved in the DB
         with data.db.conn.read_ctx() as cursor:
             # These 2 vars contain the timestamp of our side. When did this DB try to upload
-            self.last_data_upload_ts = data.db.get_setting(cursor, name='last_data_upload_ts')
+            self.last_data_upload_ts = data.db.get_cache(cursor).last_data_upload_ts
```

Comment:
could we keep the other interface for selecting the name of the key that we want? the problem with the approach of querying the object is that you do a serialization and we build all the object with all the fields when we need only one